src/main.c

```diff
@@ -221,9 +221,9 @@ char message[25][80] = /**< Array of CW messages
 			* message[1]  (F2)  - insert pressed
  			*/
 {
-    "TEST %\n", "@ DE %\n", "@ [\n", "TU 73\n", " @\n", "%\n",
-    "@ SRI QSO B4 GL\n", "AGN\n",
-    " ?\n", " QRZ?\n", " PSE K\n", "TEST % %\n", "@ [\n", "TU %\n",
+    "TEST %", "@ DE %", "@ [", "TU 73", " @", "%",
+    "@ SRI QSO B4 GL", "AGN",
+    " ?", " QRZ?", " PSE K", "TEST % %", "@ [", "TU %",
```

Comment:
Some messages here have leading space. In case the user would like to have such space (or trailing space) it should be possible to do this via config. I assume that config values will be chomp'ed, so an option would be to have a replacement character for space the same way as for NL. Underscore would a natural candidate.
Something like
```
F9 = _?
```

Reply:
After fiddling a bit with digi messages I think leading/trailing spaces are not really needed or can be effectively replaced by NL (|).
So how about

- drop leading spaces in CW messages too (incl. built-ins, here the op has more control on pacing the sending anyway)
- keep chomping messages in `parse_logcfg` and in `listmessages`
- add note in man page that unspecified digimode messages will have the value of their CW counterpart prepended with |. (`rules.c` line 87)

Reply:
Done.